Runtime/Behaviours/CinemachineCollider.cs

```diff
@@ -326,6 +328,7 @@ public override float GetMaxDampTime()
                     // Apply damping
                     if (deltaTime >= 0 && VirtualCamera.PreviousStateIsValid)
                     {
+                        // TODO: when delta time is 0, the displacement here is wrong Damper.Damp should return -extra.previousDisplacement
```

Comment:
Can you explain?  To my thinking, if deltaTime is 0 then displacement should not change - i.e. it should be the same as previousDisplacement.

Reply:
You are right. I meant for my test to work. It would need to return -extra.previousDisplacement.